ui/app/src/Footer.tsx

```diff
@@ -22,21 +24,33 @@ const style: SxProps<Theme> = {
 };
 
 export default function Footer(): JSX.Element {
+  const { response, loading, error } = useHealth();
   return (
-    <Box sx={style}>
-      <ul>
-        <li>&copy; The Perses Authors {new Date().getFullYear()}</li>
-        <li>
-          <a
-            href="https://github.com/perses/perses"
-            target="_blank"
-            rel="noreferrer"
-          >
-            <Github sx={{ verticalAlign: 'bottom' }} />
-          </a>
-        </li>
-        <li>v0.1.0</li>
-      </ul>
-    </Box>
+    <>
+      <Box sx={style}>
+        <ul>
+          <li>&copy; The Perses Authors {new Date().getFullYear()}</li>
+          <li>
+            <a
+              href="https://github.com/perses/perses"
+              target="_blank"
+              rel="noreferrer"
+            >
+              <Github sx={{ verticalAlign: 'bottom' }} />
+            </a>
+          </li>
+          <li>
+            {loading ? (
```

Comment:
Love it. This "if loading" thing is something I see in React all the time, so think you nailed it.